### ISSUE TITLE
Exclusive write access

### ISSUE THREAD

**Issue by jgrete** [nextcloud/deck#925]
We use Deck in small team of five to manage our work. We would like to have a checkbox to gain exclusive write access to a card. We often had the situation that one team member overwrites some other members changes.
Regards
Juergen

**Comment by juliushaertl** [comment]
Yes, we should definitely have some locking in place. This is also related to https://github.com/nextcloud/deck/issues/256 so the content gets updated once someone else pushes changes.

**Comment by juliushaertl** [comment]
https://fideloper.com/etags-and-optimistic-concurrency-control might be the way to go with this together with https://github.com/nextcloud/deck/issues/1650

**Comment by juliushaertl** [comment]
cc @stefan-niedermann @desperateCoder as this might be also very useful for the mobile app

**Comment by jgrete** [comment]
Nice to hear that my wish isn't lost in time and spacer :-)
